src/i18n.js

```diff
@@ -101,6 +101,11 @@ i18n
               },
             },
           },
+          projectHeader: {
```

Comment:
Atm this thing is alphabetical for future ease of finding things

Reply:
My bad! Done in [0110aad](https://github.com/RaspberryPiFoundation/editor-ui/pull/314/commits/0110aad93a16ccddd863fa4b82676619d35a587f)